recipes/cyipopt/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "cyipopt" %}
+{% set version = "0.1.7" %}
+{% set sha256 = "e62d4265743078c60f0af1426218310a9fbc1b6d0d0e60879e172d569dd3343d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/matthias-k/cyipopt/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # TODO: Once ipopt is available for Windows in conda forge, remove the
+  # following line.
+  skip: True  # [win]
+  # NOTE : Set CC to force gcc on OSX.
```

Comment:
I had to do this to make it compatible with the ipopt-feedstock which is compiled with gcc for linux and osx. Should I raise an issue there? Seems like it would need to be fixed there first. Or can I compile this with clang even though the main dependency, ipopt, is compiled with gcc? The package uses pkg-config to grab the linker info from ipopt and this contains `-lgcc_eh` which fails with clang.